money-monkey-web/app/ui/PopupModal.tsx

```
@@ -0,0 +1,57 @@
+    return () => {
+      document.body.style.overflow = "unset";
+    };
+  }, [visible]);
+
+  return (
+    <div
+      className={`transition w-screen h-screen fixed left-0 top-0 flex items-center justify-center ${
+        visible ? "bg-gray-200 bg-opacity-50" : "pointer-events-none"
+      }`}
+      onClick={onClose}
+    >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)